package/CHANGELOG

```diff
@@ -19,6 +19,7 @@ The rules for this file:
   * 0.16.0
 
 Enhancements
+  * Added unit tests for bad file mode passed to SelectionWriter
```

Comment:
Admittedly, I am not looking at the full CHANGELOG at the moment and can't remember if we recorded tests in the past. However, I would say that pure tests should not be listed in CHANGELOG because it's not really very informative for most users.

By no means does this diminish the tons of kudos to you (and anyone) who takes the time to add tests!

You should still add your name to the list of contributors of this release, of course.

My $0.02.